vector-im/element-android
# Fix rendering of edited polls (PSG-963)

## Type of change


## Content
- For an edited poll in timeline, original poll question and answers are rendered instead of edited values.
- The problem also exists in message action preview and preview in room list
- We should reflect the last edited content



## Motivation and context
Fixes #7938.



## Screenshots / GIFs





## Tests



- Create a poll, edit poll question and answers
- See the poll is rendered in timeline with edited question and values
- See the edited question is shown in message action preview
- See the edited question is shown in room list preview

## Tested devices

- OS version(s):

## Checklist
